lua/obsidian/command.lua

```diff
@@ -5,6 +5,25 @@ local echo = require "obsidian.echo"
 local util = require "obsidian.util"
 
 local command = {}
+local attach_find_command = function(client, opts)
+  -- sort_by is:
+  -- "none"
+  -- "path"
+  -- "modified"
+  -- "accessed"
+  -- "created"
+  local sort_by = client.opts.sort_by
+  if sort_by == nil then
+    return opts
+  end
+  local sort_reversed = client.opts.sort_reversed
+  local sort = "sortr" -- default sort is reverse
+  if sort_reversed == false then
+    sort = "sort"
+  end
+  opts.find_command = { "rg", "--files", "--" .. sort, sort_by }
```

Comment:
We should start with `util.FIND_CMD` command and the `--sort/--sortr` options to that.

Reply:
`find_command` is the `telescope.find_files` parameter, I don't understand it has any business with `util.FIND_CMD`